<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EspaceAdmin</class>
 <widget class="QMainWindow" name="EspaceAdmin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1278</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="deconnexion">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Deconnexion</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>1201</width>
      <height>491</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Client</string>
     </attribute>
     <widget class="QLineEdit" name="client_recherche">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>225</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>255</x>
        <y>10</y>
        <width>291</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string>Modifier Client</string>
      </property>
      <widget class="QComboBox" name="client_comboBox_modif">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>17</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>12</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>ID Client</string>
       </property>
      </widget>
      <widget class="QPushButton" name="client_ModifierBouton">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>150</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_40">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Email :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_41">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Mot de passe :</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="client_email_modif">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>50</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="client_mdp_modif">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>90</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>223</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Recherche :</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_client">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>260</y>
        <width>781</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>545</x>
        <y>10</y>
        <width>291</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string>Supprimer Client</string>
      </property>
      <widget class="QComboBox" name="client_comboBox_supp">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>17</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>12</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>ID Client</string>
       </property>
      </widget>
      <widget class="QPushButton" name="client_SupprimerBouton">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>150</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_43">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Email :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_44">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Mot de passe :</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="client_email_supp">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>50</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextBrowser" name="client_mdp_supp">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>90</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="PDF">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>220</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Parking</string>
     </attribute>
     <widget class="QTableView" name="tableView_parking">
      <property name="geometry">
       <rect>
        <x>45</x>
        <y>260</y>
        <width>781</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>10</y>
        <width>291</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string>Supprimer Parking</string>
      </property>
      <widget class="QComboBox" name="parking_comboBox_supp">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>17</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>12</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>ID Parking</string>
       </property>
      </widget>
      <widget class="QPushButton" name="parking_SupprimerBouton">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>180</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_27">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Nombre de place :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_24">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_25">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Etat :</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="parking_nom_supp">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>50</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextBrowser" name="parking_etat_supp">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>90</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextBrowser" name="parking_nbplace_supp">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>130</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLineEdit" name="parking_recherche">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>225</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>291</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string>Modifier Parking</string>
      </property>
      <widget class="QComboBox" name="parking_comboBox_modif">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>17</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>12</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>ID Parking</string>
       </property>
      </widget>
      <widget class="QPushButton" name="parking_ModifierBouton">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>170</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_28">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Nombre de place :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_29">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_30">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Etat :</string>
       </property>
      </widget>
      <widget class="QComboBox" name="parking_comboBox_etat_modif">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>90</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="parking_nbplace_modif">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>130</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="parking_nom_modif">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>50</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>223</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Recherche :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_Tri">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>230</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>251</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajouter Parking</string>
      </property>
      <widget class="QPushButton" name="parking_AjouterBouton">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>160</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_31">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Etat :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_32">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Nombre de place :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_33">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom :</string>
       </property>
      </widget>
      <widget class="QComboBox" name="parking_comboBox_etat">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="parking_nom">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>39</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="parking_nbplace">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>119</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>227</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Tri :</string>
      </property>
     </widget>
     <widget class="QLabel" name="statistiques">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>40</y>
        <width>311</width>
        <height>401</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>169</y>
        <width>151</width>
        <height>201</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>151</width>
         <height>201</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1278</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
